docs/pages/docs/01-cloud-deployment/gc-deployment.md

```diff
@@ -74,14 +53,44 @@ To check cluster health, execute the `grakn cluster status` command. The output
 ![](/images/gc-cluster-health.png)
 
 #### Accessing the Graql console
-To access the Graql console, a user password is required. You can see it in the Google console screen in the red cirlce:
+To access the Graql console, a user password is required. You can see it in the Google console screen in the red circle:
 
 ![](/images/gc-user-password.png)
 
 To log into the Graql console, simply type `graql console`. After entering the user credentials (user: grakn, password: the one from Google console) you are free to interact with Grakn via the Graql terminal. Succesful login attempt shall look like this:
 
 ![](/images/gc-graql-console.png)
 
+A summary of available commands can be found [here](http://dev.grakn.ai/docs/get-started/graql-console).
+
+#### Accessing the Grakn console
+The Grakn console can be accessed similarly to graql console by typing `grakn console start` and providing the user credentials. Successful login will look like this:
+
+![](/images/gc-grakn-console.png)
+
+Grakn console allows to perform the following actions:
+
+* create a new user:
+
+`CREATE USER username WITH PASSWORD userpassword WITH ROLE admin`
+
+* update existing user's password and/or role:
+
+`UPDATE username WITH PASSWORD newpassword WITH ROLE newrole`
```

Comment:
can we change the update section to:
```
update existing user's password

UPDATE username WITH PASSWORD newpassword

```
Let's not mention the role for now, as we only support `admin` users , until the day authorisation will be added.
Also maybe we should mention that only `admin` user will work when creating a new user. 🤔 
Sorry for not mentioning this before @kasper-piskorski